drivers/kscan/Kconfig.sdl

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2020 Jabil Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+config KSCAN_SDL
+	bool "SDL event filter for touch panel emulation"
+	depends on SDL_DISPLAY
```

Comment:
Maybe better depend on SDL (`HAS_SDL`) instead of `SDL_DISPLAY`

Reply:
Seems to that both will do the same trick. But theoretically SDL could be enabled without SDL_DISPLAY. Done